files/clappr-rtmp/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "clappr-rtmp-plugin",
+  "repo": "flavioribeiro/clappr-rtmp-plugin",
+  "files": {
+    "basePath": "dist/",
+    "include": ["rtmp.min.js", "rtmp.js", "./assets/**/*"]
```

Comment:
If you are going to include everything in `dist/` then you dont need to use the `include` parameter. `"basePath": "dist/"` is enough and will add everything that its in that folder